setup.md

```diff
@@ -36,12 +36,20 @@ Then create a new notebook using the drop-down menu on the right to select 'Pyth
 
 ![](../fig/new-notebook.png)
 
-To start the Python interpreter without the notebook, open a terminal or git bash and type the command:
+To start the Python interpreter without the notebook, open a terminal
+or command prompt and type the command:
```

Comment:
Actually, could you please also change `type the command` to `execute the command`?